Dockerfile

```diff
@@ -31,4 +31,4 @@ ENV PORT=4000
 EXPOSE $PORT
 
 # Configure the default command to build from the mounted repository.
-CMD bundle exec jekyll serve --force-polling -H $HOST -P $PORT
+CMD bundle exec jekyll serve --incremental -H $HOST -P $PORT
```

Comment:
Incremental is still marked as experimental, and the docs say it may break the site in some cases. If it's substantially more efficient, then it's fine if you leave it, but if it's only marginally beneficial, I'd drop it.